Makefile

```diff
@@ -538,6 +538,11 @@ cadence-releaser: $(BINS_DEPEND_ON)
 	$Q echo "compiling cadence-releaser with OS: $(GOOS), ARCH: $(GOARCH)"
 	$Q ./scripts/build-with-ldflags.sh -o $@ cmd/tools/releaser/releaser.go
 
+BINS  += cadence-releasetagger
+cadence-releasetagger: $(BINS_DEPEND_ON)
+	$Q echo "compiling cadence-releasetagger with OS: $(GOOS), ARCH: $(GOARCH)"
+	$Q go build -o $@ cmd/tools/releasetagger/main.go
```

Comment:
if you do end up using x/mod/semver, this is likely better as
```
$(BIN)/releasetagger: internal/tools/go.mod $(wildcard internal/tools/releasetagger/*) | $(BIN)
	$(call go_build_tool,./releasetagger)
```
as that way it'll rebuild on go / mod changes.